src/RoslynInsertionTool/RoslynInsertionTool/RoslynInsertionTool.cs

```diff
@@ -24,7 +24,7 @@ public static partial class RoslynInsertionTool
         public static readonly Guid VSRepoId = new Guid("a290117c-5a8a-40f7-bc2c-f14dbe3acf6d");
         //Easiest way to get these GUIDs is to create a PR search in AzDo
         //You'll get something like https://dev.azure.com/devdiv/DevDiv/_git/VS/pullrequests?_a=active&createdBy=GUID-here
-        public static readonly Guid MLInfraSwatUserId = new Guid("6c25b447-1d90-4840-8fde-d8b22cb8733e");
+        public static readonly Guid ReviewerUserId = new Guid($"{Options.ReviewerGUID}");
```

Comment:
If it is practical, I think it would be best to delete this field. Also, it seems like this may have wrong semantics, since this initializer will run before we set the `Options` in `PerformInsertionAsync` below.

Reply:
Yes, this was done lazily and I sent the PR before testing just in case I needed to run checks but will be cleaning that up and since I can test locally I will abandon this now until it's ready.